src/main/java/seedu/address/logic/commands/EditPropertyCommand.java

```diff
@@ -44,17 +44,17 @@
     public static final String COMMAND_WORD = "edit property";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits a property in the app. \n"
-            + "Parameters: INDEX "
-            + PREFIX_NAME + "NAME "
-            + PREFIX_TYPE + "TYPE "
-            + PREFIX_ADDRESS + "ADDRESS "
-            + PREFIX_POSTAL + "POSTAL "
-            + PREFIX_DEADLINE + "DEADLINE "
-            + PREFIX_REMARK + "REMARK "
-            + PREFIX_CLIENT_NAME + "CLIENT_NAME "
-            + PREFIX_CLIENT_CONTACT + "CLIENT_CONTACT "
-            + PREFIX_CLIENT_EMAIL + "CLIENT_EMAIL "
-            + PREFIX_CLIENT_ASKING_PRICE + "CLIENT_ASKING_PRICE \n"
+            + "Parameters: INDEX ["
+            + PREFIX_NAME + "NAME] ["
+            + PREFIX_TYPE + "TYPE] ["
+            + PREFIX_ADDRESS + "ADDRESS] ["
+            + PREFIX_POSTAL + "POSTAL] ["
+            + PREFIX_DEADLINE + "DEADLINE] ["
+            + PREFIX_REMARK + "REMARK] ["
+            + PREFIX_CLIENT_NAME + "CLIENT_NAME] ["
+            + PREFIX_CLIENT_CONTACT + "CLIENT_CONTACT] ["
+            + PREFIX_CLIENT_EMAIL + "CLIENT_EMAIL] ["
+            + PREFIX_CLIENT_ASKING_PRICE + "CLIENT_ASKING_PRICE]\n"
             + "Example: " + COMMAND_WORD + " 1 "
```

Comment:
Also, remember to add the tags parameter too

Reply:
ooooh nice thanks! added it!